vagrant/ansible/roles/samba.setup/tasks/glusterfs/main.yml

```diff
@@ -0,0 +1,12 @@
+- name: Install samba and samba-vfs-glusterfs packages
```

Comment:
I think we can use the following centos8 specific glusterfs task as common to all by moving it under _samba.setup/tasks/glusterfs/main.yml_ (similar to what we have inside _ctdb.setup/tasks/glusterfs/main.yml_)

```
- name: Selinux - Allow Samba to access glusterfs services
  seboolean:
    name: "{{ item }}"
    state: yes
    persistent: yes
  with_items:
    - samba_load_libgfapi
    - samba_export_all_rw
```

I don't completely remember the need for `samba_export_all_rw` boolean but enabling it should not harm the test environment(centos7).

Reply:
Done